plugin/system_copy.vim

```diff
@@ -6,7 +6,7 @@ let s:mac = 'mac'
 let s:windows = 'windows'
 let s:linux = 'linux'
 
-if exists('g:loaded_system_copy') || &cp || v:version < 700
+if exists('g:loaded_system_copy') || v:version < 700
```

Comment:
Can you explain this and why you are removing it? I copied this line from other plugins without fully understanding.